spec/models/report_spec.rb

```diff
@@ -576,6 +576,41 @@
         expect(row[:topic_id]).to eq(post.topic.id)
       end
     end
+
+    context "with editor filter" do
+      let(:editor_with_one_edit) { Fabricate(:user) }
+      let(:editor_with_two_edits) { Fabricate(:user) }
+
+      let(:report_with_one_edit) do
+        Report.find('post_edits', { filters: { 'editor_id' => editor_with_one_edit.id } })
+      end
+
+      let(:report_with_two_edits) do
+        Report.find('post_edits', { filters: { 'editor_id' => editor_with_two_edits.id } })
+      end
+
+      let(:posts) { Fabricate.times(3, :post) }
+
+      before do
+        freeze_time
+
+        2.times do |i|
+          posts[i].revise(editor_with_two_edits, raw: "edit #{i + 1}")
+        end
+
+        posts.last.revise(editor_with_one_edit, raw: 'edit 3')
+      end
+
+      it "returns a report for a given editor" do
+        expect(report_with_one_edit.data.count).to be(1)
+        expect(report_with_two_edits.data.count).to be(2)
+        expect(report_with_two_edits.data.count).to be(2)
+
+        expect(report_with_one_edit.filters['editor_id']).to eq(editor_with_one_edit.id)
+        expect(report_with_two_edits.filters['editor_id']).to eq(editor_with_two_edits.id)
```

Comment:
The assertion here feels a little odd to me because we seem to be testing for the arguments which we intentionally pass to the report.